src/main/java/stsjorbsmod/cards/wanderer/Mindworm.java

```diff
@@ -23,8 +23,8 @@
     private static final int COST = 0;
     private static final int DAMAGE = 6;
     private static final int UPGRADE_PLUS_DAMAGE = 1;
-    private static final int BONUS_SNAPPED_DAMAGE = 10;
-    private static final int UPGRADE_PLUS_BONUS_SNAPPED_DAMAGE = 7;
+    private static final int BONUS_SNAPPED_DAMAGE = 8;
```

Comment:
I think these updated numbers are off; from Discord,

> [10:43 PM] jorbs: i'd just go to 6 + 6/7 + 9 but i have no idea i've never actually played with it in a deck that snapped early yet

I think that would mean

```java
    private static final int BONUS_SNAPPED_DAMAGE = 6;
    private static final int UPGRADE_PLUS_BONUS_SNAPPED_DAMAGE = 3;
```

Reply:
Ah, forgot to update after jorbs commented.